CartPole/cartpole_tf.py

```diff
@@ -82,10 +90,9 @@ def Q2u_tf(Q):
     In future there might be implemented here a more sophisticated model of a motor driving CartPole
     """
     u = tf.convert_to_tensor(u_max, dtype=tf.float32) * (
-            Q +
-            tf.convert_to_tensor(controlDisturbance, dtype=tf.float32) * tf.random.normal(shape=tf.shape(Q),
-                                                                                          dtype=tf.float32) + tf.convert_to_tensor(
-        controlBias, dtype=tf.float32)
+        Q
+        + tf.convert_to_tensor(controlDisturbance * rng.standard_normal(size=Q.shape), dtype=tf.float32)
```

Comment:
I thought not. But it can be, if we want to specify it so. The controllers I've worked with compute on `float32` tensors and arrays.

What would be the advantage of `float64` over `float32`?